src/System.Threading.Tasks/pkg/aot/System.Threading.Tasks.pkgproj

```diff
@@ -7,5 +7,11 @@
     <PreventImplementationReference>true</PreventImplementationReference>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
I don't know what the placeholder is for, I just updated the one that was there for Globalization, which was referring to netcore50. The placeholder wasn't there in System.Threading.Tasks.